src/js/api/shipment.js

```diff
@@ -217,6 +217,7 @@ ripe.Ripe.prototype.createShipmentP = function(options, callback) {
  *    - 'name' - Customer's contact name.
  *    - 'phone' - Customer's contact phone.
  *    - 'address' - Customer's address.
+ *  - 'orders' - A list of RIPE Core order IDs.

```

Comment:
In some docs you have this in different ordering

Reply:
I followed the model order. References usually come last from what I can tell.

Reply:
In `createShipment' and `createShipmentP` docs you have the orders docs right after status, just saying they should be equall